galata/test/jupyterlab/inline-completer.test.ts

```diff
@@ -320,4 +320,36 @@ test.describe('Inline Completer', () => {
       expect(noGhostTextHeight).toEqual(finalHeight);
     });
   });
+  test.describe('Ghost text with Syntax Highlighting', () => {
+    test.use({
+      mockSettings: {
+        ...galata.DEFAULT_SETTINGS,
+        [PLUGIN_ID]: {
+          ...SHARED_SETTINGS,
+          showWidget: 'never',
+          ghostSyntaxHighlighting: true
+        }
+      }
+    });
+
+    test('should apply syntax highlighting', async ({ page }) => {
+      const cellEditor = (await page.notebook.getCellInputLocator(2))!;
+      // Type 'sug' which matches 'suggestion_1 = 1' from the setup notebook.
+      await page.keyboard.type('ug');
```

Comment:
Let's match what comment says.
```suggestion
      // Type 'sug' which matches 'suggestion_1 = 1' from the setup notebook.
      await page.keyboard.type('sug');
```